modules/numlock.nix

```
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.xsession.numlock;
+in
+  {
+    options = {
+      xsession.numlock.enable = mkEnableOption "Numlock";
```

Wikipedia calls the key "Num Lock" so can have
```suggestion
      xsession.numlock.enable = mkEnableOption "Num Lock";
```